charts/shoot-core/components/charts/node-problem-detector/Chart.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: v1
+name: node-problem-detector
+version: "0.7.1"
```

Shoudn't it be "1.6.1" as per https://github.com/helm/charts/blob/master/stable/node-problem-detector/Chart.yaml#L3
```suggestion
version: "1.6.1"
```